### PR TITLE
[16.0][MIG] dms_user_role: Migration to 16.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 16.0

Please @chienandalu and @carolinafernandez-tecnativa can you review it?

@Tecnativa

### CODE REVIEW

File: dms_user_role/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#. Go to `Settins / Users & Companies / Roles` and create a new one.
```
Comment (chienandalu) [inline dms_user_role/readme/USAGE.rst]:
```suggestion
#. Go to `Settings / Users & Companies / Roles` and create a new one.
```

**Review by chienandalu — COMMENTED** [review]
:+1: Just a tiny typo:

**Review by chienandalu — APPROVED** [review]

**Review by carolinafernandez-tecnativa — APPROVED** [review]
Functional review LGTM :) Thanks!

**Pull request merged**